sw/device/tests/crypto/cryptotest/firmware/BUILD

```diff
@@ -176,6 +199,7 @@ opentitan_test(
     deps = [
         ":aes",
         ":aes_sca",
+        ":hash",
```

Comment:
Needing to add the firmware handlers for each algorithm here is necessary for the build to pass CI, and we cannot replicate this change when merging to `master` because the `chip_pen_test` build is local to this branch. Unfortunately, I think this means that we cannot automatically cherry-pick future cryptotest PRs to the `earlgrey_es_sival` branch. This will require a PR every time, or at least another glob PR like this one.